torch/meta_registrations.py

```diff
@@ -0,0 +1,44 @@
+import torch
+
+Library = torch.library.Library
+aten = torch.ops.aten
+
+meta_lib = Library("aten", "IMPL", "Meta")
+
+# Implementations below are taken from https://github.com/albanD/subclass_zoo/blob/main/python_meta_tensor.py
+def meta_index_select(*args, **kwargs):
+    self, dim, index = args
+    assert not kwargs
```

Comment:
Don't have to take these in varargs anymore, no?

Reply:
that's true. In case of abs, where more than one overloads (default, out) don't have meta support, maybe we could use varargs (handle kwargs) and register the same function for multiple overloads.

Reply:
IME, for meta it's better to register one function per overload